### ISSUE TITLE
Install failure report for v1.0.0

### ISSUE THREAD

**Issue by Graham-72** [backdrop/backdrop-issues#590]
Trying to install on a VPS with PHP5.5 I get this on screen

An AJAX HTTP error occurred. HTTP Result Code: 200 Debugging information follows. Path: http://www.oliverweb.net/backdrop/core/install.php?langcode=en&profile=standard&id=1&op=do_nojs&op=do StatusText: OK ResponseText: Home | Backdrop @import url("http://www.oliverweb.net/backdrop/core/misc/normalize.css?"); @import url("http://www.oliverweb.net/backdrop/core/modules/system/css/system.theme.css?"); @import url("http://www.oliverweb.net/backdrop/core/modules/system/css/system.css?"); @import url("http://www.oliverweb.net/backdrop/core/modules/user/css/user.css?"); @import url("http://www.oliverweb.net/backdrop/core/modules/system/css/system.admin.css?"); @import url("http://www.oliverweb.net/backdrop/core/modules/system/css/system.maintenance.css?"); @import url("http://www.oliverweb.net/backdrop/core/themes/seven/style.css?"); @import url("http://www.oliverweb.net/backdrop/core/themes/seven/seven.base.css?"); Installation tasksStep 5 of 6Choose language(done)Choose profile(done)Verify requirements(done)Set up database(done)Install profile(active)Configure site Home SQLSTATE[42S02]: Base table or view not found: 1146 Table 'bkdrop.watchdog' doesn't exist

and a very long list of errors starts with:

Warning: array_map(): Argument #2 should be an array in ConfigFileStorage->listAll() (line 1003 of /var/www/vhosts/oliverweb.net/httpdocs/backdrop/core/includes/config.inc).
Warning: Invalid argument supplied for foreach() in _node_types_build() (line 656 of /var/www/vhosts/oliverweb.net/httpdocs/backdrop/core/modules/node/node.module).
Warning: array_map(): Argument #2 should be an array in ConfigFileStorage->listAll() (line 1003 of /var/www/vhosts/oliverweb.net/httpdocs/backdrop/core/includes/config.inc).
Warning: Invalid argument supplied for foreach() in field_read_fields() (line 331 of /var/www/vhosts/oliverweb.net/httpdocs/backdrop/core/modules/field/field.crud.inc).
Warning: array_map(): Argument #2 should be an array in ConfigFileStorage->listAll() (line 1003 of /var/www/vhosts/oliverweb.net/httpdocs/backdrop/core/includes/config.inc).
Warning: Invalid argument supplied for foreach() in field_read_fields() (line 331 of /var/www/vhosts/oliverweb.net/httpdocs/backdrop/core/modules/field/field.crud.inc).

The above was on PHP 5.5.9 with mySQL 5.5.40
I have since installed without problems on another VPS using PHP 5.5.7 and mySQL 5.1.72

**Comment by tarekdj** [comment]
Install working 100% on: 
* Apache version : Apache/2.4.10 (Win32) OpenSSL/1.0.1h PHP/5.5.20
* Apache version : Apache/2.4.10 (Win32) OpenSSL/1.0.1h PHP/5.6.4

**Comment by quicksketch** [comment]
Thanks @Graham-72 for the report! We worked out most of the install issues recently in https://github.com/backdrop/backdrop-issues/issues/531.

The *one* install issue we haven't been able to reproduce is a table going missing during the installer. In the meta issue, the error was:

> I would sometimes get this message during an installation: "SQLSTATE[42S02]: Base table or view not found: 1146 Table 'backdrop_1.cache_filter' doesn't exist"

But in this report it loos like it's watchdog that failed:

> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'bkdrop.watchdog' doesn't exist

I'm not sure if that's just coincidence or the same problem manifesting itself. Please let us know if you can get Backdrop working on that VPS, we'd like to figure out the cause.

**Comment by Graham-72** [comment]
I have solved my problem by installing an alternative and newer version of PHP on my VPS (PHP 5.5.20). This has a different version of the mysql libraries and I now have a site working at www.oliverweb.net/backdrop where I plan to add notes about my ongoing trial of Backdrop.

I wonder why Backdrop didn't like my 'out-of-the-box' Plesk/PHP configuration? Drupal 7 seems happy with it, perhaps I should try Drupal 8 alpha?

**Comment by klonos** [comment]
http://www.oliverweb.net/backdrop/about-me

> ...So here goes, with what at the moment I think of as Drupal 7.5 !

Wanted to say the very exact thing for so long now.

**Comment by Graham-72** [comment]
I have now tried installing Drupal 8 beta on my VPS in a domain that uses PHP5.5.9 and it was successful, and I had no mysql problem. So there must be some special requirement in Backdrop, not inherited from Drupal???

**Comment by quicksketch** [comment]
> I have now tried installing Drupal 8 beta on my VPS in a domain that uses PHP5.5.9 and it was successful, and I had no mysql problem. So there must be some special requirement in Backdrop, not inherited from Drupal???

Backdrop is much more like Drupal 7 than 8, so most things we "inherit" come from D7. Apparently D7 can't currently install on PHP 5.5 with an OPCode cache enabled, which we've fixed, but there must be some other PHP 5.5.x issues that haven't been figured out yet. Thanks again for posting your experience, we'd love to figure out what these lingering problems are caused by.

> This has a different version of the mysql libraries and I now have a site working at www.oliverweb.net/backdrop where I plan to add notes about my ongoing trial of Backdrop.

That's great! Thanks so much for taking the time to evaluate Backdrop. :smile:

**Comment by Graham-72** [comment]
Thanks very much for responding. I have done a bit more testing on my VPS and I find that when using the PHP 5.5.9 build Zend OPcache is present and enabled and, as you say, D7 will not install, though D8 (beta v4) will install OK. But you say you have fixed this for Backdrop, so maybe I was hitting some other problem. I have not got OPcache in my PHP 5.5.20 build, and have no problems.

**Comment by quicksketch** [comment]
Hm, looks like we need more investigation to see if our OPCache problem is completely solved. Thanks @Graham-72 for your info!

**Comment by rolturn** [comment]
Had the same issue on PHP 5.4 and upgrade resolved it. As a designer I don't know what the development issues are, but since upgrading PHP version seemed to resolve the issue may I suggest changing System Requirements on https://backdropcms.org/requirements to a higher version of PHP. That may resolve confusion in those of us less knowledgeable in that area. I almost abandoned testing  Backdrop.

**Comment by Graham-72** [comment]
My feeling, without yet reviewing the evidence, is that it may not be sufficient to say which version of PHP is required, but we need to identify the detailed configuration requirements for PHP. My past problems were sometimes with PHP installations that would be OK for Drupal 7 but not for Backdrop. When I have some time I could review my experiences and try and pinpoint what made a difference. I still have about 4 different PHP versions available as alternatives on my VPS, and I could adopt a more scientific approach and change just one variable at a time, rather than change both PHP version and configuration in order to get Backdrop installing. I would be very happy to share information on the configurations I have tried so far.

**Comment by klonos** [comment]
Related?: #1072

**Comment by Graham-72** [comment]
I am trying to move a working Backdrop site from my server to the client's current hosting service and it will not yet install on this other server. We are working on it and if successful I will report back here on our findings.
The interesting thing is that Drupal 7 will install successfully on this server. Do we have a list of the different hosting requirements for Backdrop?

**Comment by klonos** [comment]
@Graham-72 ?? How did it go with this? Can we close it?

**Comment by Graham-72** [comment]
@klonos I had to abandon the attempt without finding out what the problem was. The site is OK and still hosted on my server. It is disappointing not to have an answer but it doesn't seem to be a common occurrence, so closing this is fine by me. Thanks for asking.

**Comment by olafgrabienski** [comment]
> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'bkdrop.watchdog' doesn't exist
> (...)
> The above was on PHP 5.5.9 with mySQL 5.5.40

I just saw the very same message about the missing *watchdog* table when I tried to install Backdrop 1.8.0 on a client's server without success. PHP is 5.3.3, mySQL 5.1.73. It doesn't matter if OPcache is enabled or not: wasn't able to install Backdrop anyway.

Interesting: I installed Backdrop on my own webspace and was able to transfer the site to the client's server where it is running smoothly. So this issue seems to be 'only' related to the installation process.
